site/soc/software/charts/osh/openstack-compute-kit/neutron.yaml

```diff
@@ -40,33 +34,12 @@ data:
     pod:
       replicas:
         server: 1
-    labels:
-      agent:
-        l3:
-          node_selector_key: openstack-control-plane
-          node_selector_value: enabled
     network:
+      auto_bridge_add:
+        br-ex: {{ neutron_external_interface }}
       interface:
-        tunnel: docker0
+        tunnel: {{ neutron_tunnel_device }}
     conf:
-      neutron:
-        DEFAULT:
-          l3_ha: False
-          max_l3_agents_per_router: 1
-          l3_ha_network_type: vxlan
-          dhcp_agents_per_network: 1
-      plugins:
-        ml2_conf:
-          ml2_type_flat:
-            flat_networks: public
-        openvswitch_agent:
-          agent:
-            tunnel_types: vxlan
-          ovs:
-            bridge_mappings: public:br-ex
-        linuxbridge_agent:
-          linux_bridge:
-            bridge_mappings: public:br-ex..
```

Comment:
Why are L52-L69 removed?

Reply:
> Why are L52-L69 removed?

because they are provided in the global definition. The removed lines are more settings for dev environment.